main/active_cname.js

```diff
@@ -119,6 +119,8 @@ module.exports = {
   'preview.spicydevs': 'spicydevs.github.io',
   'minecraft': 'mcjs.pages.dev',
   'servers': 'opaayush.github.io',
+  'nelophine': 'opaayush.github.io',
+  'uptime': '872b6773-e225-46b7-b6b9-25747d831c64.id.repl.co',
```

Comment:
make others confusing

Reply:
Ok but pls merge fast what should I do?
All the domains i own i seperated them together

Reply:
why not 3rd-level domains like uptime.yourname.js.cool, servers.yourname.js.cool

Reply:
@OpAayush